data/config-schema/src/main/resources/net/corda/schema/configuration/rest/1.0/corda.rest.json

```diff
@@ -62,6 +62,13 @@
           "default": 30000
         }
       }
+    },
+    "flowStatusCleanupTime": {
```

Comment:
I feel like milliseconds is too fine a unit to use here.

Assuming we keep milliseconds, should we add 'Ms' to the end of the name, like `idleTimeoutMs` above?

Reply:
I agree that milliseconds probably allows a lot more granularity than we need, but I think I'll keep it now for a couple of reasons:
1. All other config values in this section use milliseconds, so it's predictable;
2. We suggest in the description that it must be set higher than another config, which is defined in ms, so I don't want to make users do the maths.

We can get this config up to about 24 days using milliseconds; if there comes a point that we need a longer retention period, I think we should change it.